tools/src/test/java/org/apache/kafka/tools/streams/DescribeStreamsGroupTest.java

```diff
@@ -180,8 +180,8 @@ public void testDescribeStreamsGroupWithStateAndVerboseOptions() throws Exceptio
     public void testDescribeStreamsGroupWithMembersOption() throws Exception {
         final List<String> expectedHeader = List.of("GROUP", "MEMBER", "PROCESS", "CLIENT-ID", "ASSIGNMENTS");
         final Set<List<String>> expectedRows = Set.of(
-            List.of(APP_ID, "", "", "", "ACTIVE:", "0:[0,1];"),
-            List.of(APP_ID, "", "", "", "ACTIVE:", "1:[0,1];"));
+            List.of(APP_ID, "", "", "", "ACTIVE:", "0:[1];", "1:[1];"),
+            List.of(APP_ID, "", "", "", "ACTIVE:", "0:[0];", "1:[0];"));
```

Comment:
this is confirming the `subtopology_partition` task ids right?

Reply:
yes